balboa-http/src/main/scala/com/socrata/balboa/server/MetricsServlet.scala

```diff
@@ -69,15 +61,12 @@ class MetricsServlet extends JacksonJsonServlet
       return unacceptable.result
     }
 
-    val begin = System.currentTimeMillis()
-
     val range = DateRange.create(period, ServiceUtils.parseDate(date).getOrElse {
       contentType = json
       return malformedDate(date).result
     })
 
-    try
-    {
+    timer("period queries") {
```

Comment:
The timers are (I believe) intended to record how long it takes to query Cassandra and send back the results, not including the time necessary to parse the request. This is the pattern followed by the two other timer instances, and I believe we should keep consistent behavior between the timers.

Additionally, in the case that the date cannot be parsed, the API would return before querying Cassandra, causing timer entries far shorter than those that actually executed a query. Not only would this cause great variance in our timer, but it wouldn't match the behavior of the old system. In the old version, when a date fails to parse it returns immediately without updating the timer.